### DISCUSSION TITLE
VERSION file populated with Unknown

### DISCUSSION THREAD

**Discussion by Johan-Peralta** [aces/Loris#9285] [Q&A] [answered]
Hi, does anybody know how to get the LORIS version properly recorded in the /var/www/loris/VERSION file?

My installations are getting Unkown in the VERSION file.

Thanks in advance for any help with the issue,
Johan

**Comment by Johan-Peralta** [comment]
Hi, update on this issue/concern.

I installed LORIS V26, and the VERSION file was populated with **v25.0.2-198-gf3034e39e**

Thanks in advance for any help,
Johan

**Reply by Johan-Peralta** [reply]
Hi, it will be helpful if the LORIS application shows the running version.

**Comment by ridz1208** [comment, marked as answer]
Hi @Johan-Peralta The release has not been published yet so it's technically not 26 yet. the version will be updated before the official release.

Thanks
